operations/deployment-engineering/frequently-asked-questions.md

```diff
@@ -0,0 +1,66 @@
+
+## Who owns long-term maintenance of solutions built by this team?
+
+Most solutions should not require long-term maintenance, as they will be standalone scripts, plugins or other code that is not dependent on Mattermost server versions.
+
+For solutions that require maintenance, we will discuss ownership between Deployment Engineering, TAMs, R&D and a vendor on a case-by-case basis.
+
+## How does this role differ from TAMs (Technical Account Managers)?
+
+TAMs are responsible for ongoing relationship management with technical champions, and own pre-defined customer onboarding in standard deployment environments (ie. follow our gold standard deployments).
+
+On the other hand, Deployment Engineering supports non-standard deployments with complex technical problems, as well as work closely with customers on temporary one-off custom solutions & integrations based on [requirements outlined by the CSMs/TAMs](https://docs.google.com/document/d/1EbP4Ab7N2hEAv--DRCcZHk0XtlG1MFP8F8F0viaTM7I/edit).
+
+Thus, Deployment Engineering acts as an extension of R&D supporting post-sales and customer success on the most complex, technical problems, with expertise on how to run and deploy Mattermost and the platform as a whole.
+
+Finally, we encourage a representative from TAM to join weekly Deployment Engineering meetings, and vice versa, to help deepen collaboration & understanding of each team’s workstreams.
+
+## Why does this role not report to Customer Success, like TAMs?
+
+TAMs are responsible for ongoing relationship management, and are evaluated against renewal rates.
+
+Deployment Engineering engages with customers on a short-term basis, in a technical capacity. Their evaluation is not based on renewal rates, and hence do not directly report to the revenue organization (Sales & Customer Success).
+
+## How does this role differ from Support Engineers & Customer Reliability Engineers?
+
+Support & Customer Reliability Engineers provide first-line of support & troubleshooting, and may fix bugs & security issues, while Deployment Engineering is a point of contact for critical deployment escalations only for customers over $50k ARR (e.g. performance degradation).
+
+## Why does this role not report to Customer Support, if they work on escalations?
+
+Support Engineers are evaluated against CSAT scores and SLA coverage. Deployment Engineering is evaluated against Customer Seats Deployed.
+
+Moreover, Deployment Engineering is not a support-function, but instead their primary ownership is to help customers run and deploy Mattermost at scale.
+
+## Will this team be a call or SET rotation?
+
+No.
```

Comment:
Aren't large upgrades happening on weekends? I understand that there is not going to be a regular on-call, but I'd expect DE to be on call during those upgrade windows

Reply:
True. Though I consider that working days on the weekend more-so than a formal on-call rotation

Reply:
> True. Though I consider that working days on the weekend more-so than a formal on-call rotation

Jason, probably this might be a longer run thing and really something that should be open to discussion, but I do believe there should be some alert/pager duty(not necessarily on-call) or some other solution to alert a CS/DE engineer if things really go south/ we encounter a sev1 outage kind of issue during odd hours (non-US/EU timings or on weekend) and the issue goes beyond the CRE scope during the triaging.                                                Case in point was a recent case with Bofa (which surfaced during apac hours and the one caused by DR job bringing down their MM instance) where I could see no alerting mechanism for other customer facing teams. We might lose out some valuable response time in case we have to reach out to other customer facing teams (thankfully we were to able to triage and do an initial investigation on that case). Again, this is just a headsup thing and doesn't have to be implemented immediately.